reference/tax.v3.yml

```
@@ -207,4 +207,20 @@ components:
                 Describes whether the stored credentials are considered complete and configured, ready to be used for Tax Provider API requests.
 
                 Merchants may enable any **configured** tax provider for storefront tax quotation.
+            target:
+              type: object
+              description: A checkout session's active tax provider connection will be determined by the target data.
```

🍹  This doesn't read well to me, the mention of a checkout session kinda threw me off. IMO something a little more direct might be easier to read? Trying to take the perspective of any of the tax integrators we work with.

I like the descriptions for `country_codes` and `subdivision_codes`, so something like
```suggestion
              description: The countries and subdivisions in which this tax provider is active
```

---

Yeah, good call - I think something more straight forward will work better here.

One thing is that automatic tax providers are inherently only active in the checkout context, this doesn't really need to be described here.